content/en/blog/2024/bitnami-using-notation.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Bitnami now uses Notation for signing and verifying images on DockerHub
```

Comment:
```suggestion
title: Bitnami now uses Notation for signing and verifying images and Helm Charts on Docker Hub
```

Reply:
To align this title with https://github.com/notaryproject/notaryproject.dev/pull/396/files#diff-40a9baefb9102a034f95dbfda47c8038ba64e5c4834ad6d0224f385ecba42d0dR14